pkg/formatted/color.go

```diff
@@ -64,6 +64,8 @@ func DecorateAttr(attrString, message string) string {
 		return fmt.Sprintf("💌 ")
 	case "taskruns":
 		return fmt.Sprintf("🗂  ")
+	case "sidecars":
```

Comment:
It's gotta be a car, right? 🤷‍♂

Reply:
Hahah it has to ! 🚗 🚗 🚗

Reply:
or mothercycle :D but the tiny red car is cutter imho!!